### PR TITLE
Update Cisco SB

### PR DESCRIPTION
**Pull request by DavidLiedke**

### CODE REVIEW

**Comment by inphobia** [comment]
i'm a heavy user of these devices & will review this change asap

**Comment by inphobia** [comment]
this one is a bit thougher, since both ciscosb/lldpextv2dot3.mib & rfc/LLDP-EXT-DOT3-V2-MIB.txt define several of the same leaf names but with different definitions.
it seems the one you included is a more recent and the one we have in rfc/ is obsolete (but that's the most recent one they have on the ieee website).

granted, the module identites are different:
ieee8023lldpV2Xdot3MIB for ciscosb/lldpextv2dot3.mib
lldpV2Xdot3MIB for rfc/LLDP-EXT-DOT3-V2-MIB.tx

but so was the case for IF-MIB.txt vs RFC1213-MIB.txt, and that resulted in https://github.com/netdisco/snmp-info/issues/325

and it's not that the ieee feedback made it any more clear to me:
https://www.ieee802.org/3/be/email/msg00253.html

so, bottom line, i'm not quite sure how to go on with this....


(and if these came from the 350-550 sb series, they released a new mib pack yesterday 😢 - Managed Switch MIB - 2.5.7.85)

**Comment by JeroenvIS** [comment]
Not a full review yet, but general comment: the EXTRAS/reports/all has a lot more changes than the EXTRAS/reports/ciscosb file. This is not a good sign. Also, the conflicting lldpV2Xdot3MIB versions are an issue.

I suggest to split this up in smaller commits. First add the Cisco specific MIBs that add new functionality / objects without depending on a Cisco specific copy of an RFC/IEEE MIB - if possible, add as few MIBs as possible per commit. We can accept those quickly. The remainder will be the subset of MIBs that somehow introduce conflicts; that will be easier to address when the total number of changes is smaller.

**Comment by inphobia** [comment]
i had a closer look but seems i didn't report back...

it seems there were more updates to those mibs, but not all changed files have a  new 'LAST-UPDATED' or 'REVISION' which might make prepmibs skip several updated mibs.

what was the source package for these mibs, where they from sb 300/500 series or the newer 350/550 series?

**Comment by inphobia** [comment]
it seems we recently started using sb350 series. i have a netdisco-mibs release planned soon, ciscosb will most likely be the last major update i'll finish before releasing.

**Comment by inphobia** [comment]
> Also, the conflicting lldpV2Xdot3MIB versions are an issue.

i noticed this and will make a new issue for this. it seems the ietf transfered ownership of several modules to the ieee,

this results in, among other things, a split between for example ieee8023lldpV2Xdot3MIB  & lldpV2Xdot3MIB.
https://www.ieee802.org/1/files/public/MIBs/LLDP-EXT-DOT3-V2-MIB-200906080000Z.mib
https://www.ieee802.org/3/1/public/mib_modules/20130411/802dot3dot1C5mib.txt
